src/gui/static/src/app/components/pages/wallets/create-wallet/create-wallet.component.ts

```diff
@@ -32,11 +32,12 @@ export class CreateWalletComponent implements OnInit {
   }
 
   createWallet() {
+    this.button.resetState();
     this.button.setLoading();
 
     const password = this.encrypt ? this.form.value.password : null;
     this.walletService.create(this.form.value.label, this.form.value.seed, this.scan, password)
-      .subscribe(() => this.dialogRef.close());
+      .subscribe(() => this.dialogRef.close(), e => this.button.setError(e));
```

Comment:
Would be better if the message is cleaned with `parseResponseMessage` from src/app/utils/index.ts, added in https://github.com/skycoin/skycoin/pull/1293

Reply:
That should happen inside the setError method 
https://github.com/skycoin/skycoin/blob/76a82204b6350a784194976f47673f9e38fc7c47/src/gui/static/src/app/components/layout/button/button.component.ts#L34

Reply:
You're right, I did not notice it because the pr did not include that change.